src/routes/blogs/olive-shared-cache/+page.svx

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Enhance team collaboration during AI model optimization with the Olive Shared Cache feature'
+date: '30th October, 2024'
+description: 'Learn how to use the shared cache feature in Olive to enhance team collaboration when optimizing AI models'
+keywords: 'GenAI , LLM, ONNXRuntime, ORT, Phi, DirectML, Windows, phi3, phi-3, llama-3.2, ONNX, SLM, edge, gpu'
+authors:
+  [
+    'Xiaoyu Zhang',
+    'Devang Patel',
+    'Sam Kemp'
+    ]
+authorsLink:
+  [
+    'https://www.linkedin.com/in/xiaoyu-zhang/',
+    'https://www.linkedin.com/in/devangpatel/',
+    'https://www.linkedin.com/in/samuel-kemp-a9253724/'
+   ]
+image: 'https://iili.io/d95Pwcx.png'
+imageSquare: 'https://iili.io/d95Pwcx.png'
+url: 'https://onnxruntime.ai/blogs/olive-shared-cache'
+---
+
+
+## 👋 Introduction
+
+In the ever-evolving realm of machine learning, optimization stands as a crucial pillar for enhancing model performance, reducing latency, and cutting down costs. Enter Olive, a powerful tool designed to streamline the optimization process through its innovative shared cache feature. 
+
+Efficiency in machine learning not only relies on the effectiveness of algorithms but also on the efficiency of the processes involved. Olive’s shared cache feature – backed by Azure Storage - embodies this principle by seamlessly allowing intermediate models to be stored and reused within a team, avoiding redundant computations.
+
+This blog post delves into how Olive’s shared cache feature can help you save time and costs, illustrated with practical examples.
+
+
+## 🤝 Team collaboration during optimization process
+
+User A begins the optimization process by employing Olive’s quantize command to optimize the [Phi-3-mini-4k-instruct](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct) model using the AWQ algorithm. This step is marked by the following command line execution:
+
+```bash
+olive quantize \
+  --model Microsoft/Phi-3-mini-4k-instruct \
+  --algorithm awq \
+  --account-name {AZURE_STORAGE_ACCOUNT} \
```

Comment:
Maybe the format of azure storage account / storage container name can be more specific? Not a hardcore azure user so not 100% sure how I'd access - maybe links to docs?